var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,29 +34,46 @@ class Relion(CMakePackage, CudaPackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     git      = "https://github.com/3dem/relion.git"
 
+    version('3.0_beta',
+            git='https://bitbucket.org/scheres/relion-3.0_beta.git')
+    version('2.1', preferred='true')
+    version('2.0.3')
```

Comment:
Does this still work if you don't specify the tag? It isn't going to automatically infer where to download `2.0.3`.

Reply:
Yes works for `2.0.3`

`Cloning git repository: https://github.com/3dem/relion.git`

Reply:
Adding tag='2.0.3' back while also removing 'develop' version as it no longer exists.